Source/Scene/Cesium3DTileset.js

```diff
@@ -222,6 +222,8 @@ define([
 
         this._readyPromise = when.defer();
 
+        this._classificationType = options.classificationType;
```

Comment:
Is this in the reference doc?

Reply:
Also include the limitations in the reference doc.  It might be easiest to add this once in the property getter and then reference it from the constructor function.